From eca1a7ffb9db8864c1ba1cca0862374137dd6ccd Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 24 Sep 2025 10:30:46 +0200
Subject: [PATCH] arm64: defconfig: addition of stm32 debug bus in fragment-02

The stm32 debug bus support is needed to handle the probe of debug-related
peripherals. Enable CONFIG_STM32_DBG_BUS as module by default on ARM64
platforms.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I8c4627cef04f06cd53b7e4247610e809aaffdcee
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/493431
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -102,6 +102,7 @@ CONFIG_SPI_STM32_OSPI=y
 CONFIG_STM32MP_PM_DOMAIN=y
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
+CONFIG_STM32_DBG_BUS=m
 CONFIG_STM32_DMA3=y
 CONFIG_STM32_FMC2_EBI=y
 CONFIG_STM32_HYPERBUS=y
